lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -79,12 +79,20 @@ def self.get_user_build_version(version:, message:)
       # @!group Documentation
       #####################################################
 
+      def self.category
+        :deprecated
+      end
+
+      def self.deprecated_notes
+        'This action is deprecated and will be removed in an upcoming Release Toolkit version. Any necessary steps that are included in this precheck action should be added directly in a repo\'s Fastfile. See https://github.com/wordpress-mobile/release-toolkit/issues/576'
+      end
+
       def self.description
-        'Runs some prechecks before preparing for a new test build'
+        '(DEPRECATED) Runs some prechecks before preparing for a new test build'
```

Comment:
Appreciate the effort here, but FWIW I wouldn't consider it necessary. Fastlane already prints the deprecation alert when the lane is called and in the docs:

![image](https://github.com/user-attachments/assets/f3fa4649-a6db-4d58-b62a-c1814e0ab873)

Reply:
Good point. Removed in https://github.com/wordpress-mobile/release-toolkit/commit/8d87d87d72a9aef1ab10dd0300fd563921d278fd